make/common/native/CompileFile.gmk

```diff
@@ -236,7 +236,7 @@ define CreateCompiledNativeFileBody
             # For assembler calls just create empty dependency lists
 	    $$(call ExecuteWithLog, $$@, $$(call MakeCommandRelative, \
 	        $$($1_COMPILER) $$($1_FLAGS) \
-	        $(CC_OUT_OPTION)$$($1_OBJ) -Ta $$($1_SRC_FILE))) \
+	        $(CC_OUT_OPTION)$$($1_OBJ) $$($1_SRC_FILE))) \
```

Comment:
Have you verified that removing `-Ta` works for the x86 assembler?

Reply:
Is this really so? The [documentation](https://learn.microsoft.com/en-us/cpp/assembler/masm/ml-and-ml64-command-line-reference?view=msvc-170) states:

```
/Ta filename 	Assembles source file whose name doesn't end with the .asm extension.
```

We want the compiler to actually do what we tell it to do, even if the files are named .S. 

Maybe this behavior has been changed in later versions, but I am a bit weary about removing it unless we have some official confirmation that `/Ta` is a no-op, and/or information on which version this started to happen.